contrib/linux/dosbox-staging.desktop

```diff
@@ -54,11 +54,11 @@ GenericName[vi]=Trình mô phỏng DOS
 GenericName[wa]=Èn emulateu DOS
 GenericName[zh_CN]=DOS 模拟器
 GenericName[zh_TW]=DOS 模擬器
-Comment=A modernized DOSBox
+Comment=DOS/x86 emulator focusing on ease of use
 Exec=dosbox
 Icon=dosbox-staging
 Type=Application
 Terminal=false
 Keywords=dos;gaming;game;games;emulator;
-Categories=Game;Emulator;
+Categories=Emulator;
```

Comment:
I'm using the traditional KDE Plasma Application Menu, and removing `Game` moves the menu entry from Games to Lost & Found. Can you please add it back?

Reply:
Sure :)